sbt-coursier/src/sbt-test/sbt-coursier/classifiers/build.sbt

```diff
@@ -0,0 +1,18 @@
+scalaVersion := "2.11.8"
+
+libraryDependencies += ("org.jclouds.api" % "nova" % "1.5.9")
+  .classifier("tests")
+
+classpathTypes += "test-jars"
+
+resolvers ++= Seq(
+  MavenRepository("Maven Central", "http://central.maven.org/maven2/")
```

Comment:
From Last-Modified, ETag, Fastly-Debug-Digest, etc they look to be the same thing.

```bash
09:37:23 $ curl -I http://central.maven.org/maven2/
HTTP/1.1 200 OK
Last-Modified: Thu, 25 May 2017 14:44:47 GMT
ETag: "0d09a809c6c2773b92e1dfe48509cb38"
Content-Type: text/html
Via: 1.1 varnish
Fastly-Debug-Digest: 9648f889e542c791069a3712497fbde4749f85854bc0f9a7409e5b5c0130957f
Content-Length: 110238
Accept-Ranges: bytes
Date: Tue, 04 Jul 2017 08:37:35 GMT
Via: 1.1 varnish
Age: 3433857
Connection: keep-alive
X-Served-By: cache-iad2150-IAD, cache-lhr6333-LHR
X-Cache: MISS, HIT
X-Cache-Hits: 0, 4
X-Timer: S1499157456.894148,VS0,VE0


09:37:36 $ curl -I http://repo1.maven.org/maven2/
HTTP/1.1 200 OK
Last-Modified: Thu, 25 May 2017 14:44:47 GMT
ETag: "0d09a809c6c2773b92e1dfe48509cb38"
Content-Type: text/html
Via: 1.1 varnish
Fastly-Debug-Digest: 9648f889e542c791069a3712497fbde4749f85854bc0f9a7409e5b5c0130957f
Content-Length: 110238
Accept-Ranges: bytes
Date: Tue, 04 Jul 2017 08:37:40 GMT
Via: 1.1 varnish
Age: 3433861
Connection: keep-alive
X-Served-By: cache-iad2150-IAD, cache-lhr6330-LHR
X-Cache: MISS, HIT
X-Cache-Hits: 0, 6
X-Timer: S1499157460.454570,VS0,VE0
```